src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,52 @@
-import React from 'react';
+import {
+  FC, useCallback, useEffect, useState,
+} from 'react';
 import { Loader } from '../Loader';
+import { ErrorMessage } from '../ErrorMessage';
+import { User, Todo } from '../../types';
+import { getUser } from '../../api';
+
+type Props = {
+  todo: Todo,
+  onClose: () => void,
+};
+
+export const TodoModal: FC<Props> = ({ todo, onClose }) => {
+  const [user, setUser] = useState<User | null>(null);
```

Comment:
the same with user, do you need to store object in state?

Reply:
i do, i use its properties in render